index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div>
```

Comment:
- It's better to use semantic tags, such as - section for the different sections of the body, article in places of divs, and so on.

Reply:
- Great Job using inline comments ✔️ 

- It is important for your review comments to be as polite as possible. I believe it will be better if you had rephrased this entire comment to be like this 👉🏽  (" I think It's better to use semantic tags, such as - section for the different sections of the body, article in places of divs, and so on because semantic tags  . . . . . . ") 
- It will be great if you add some emojis after each comment to make your suggestions sound a bit more friendly. 🌸 

Why don't we rephrase your comment to what I have just suggested?? 🤗